src/test/scala/mesosphere/marathon/api/v2/json/V2AppDefinitionTest.scala

```diff
@@ -49,11 +49,13 @@ class V2AppDefinitionTest extends MarathonSpec with Matchers {
     shouldViolate(app, "id", idError)
 
     app = app.copy(id = "-dash-disallowed-at-start".toRootPath)
-    validateJsonSchema(app, false)
+    // The JSON schema does allow this. It is not as strict as our internal validation to keep the regular
+    // expression reasonably simple.
```

Comment:
We have group support in the cli! For Universe maybe in the future, but no plans atm.